Apps/Playground/App.tsx

```diff
@@ -50,6 +50,15 @@ const EngineScreen: FunctionComponent<ViewProps> = (props: ViewProps) => {
               }
             }
```

Comment:
Will you just factor out this part to a local const lambda?
```ts
const handlePointerInput = (inputIndex: PointerInput, previousState: Nullable<number>, currentState: Nullable<number>) => {
  if (inputIndex === PointerInput.Horizontal &&
      currentState && previousState) {
        rootNode.rotate(Vector3.Down(), (currentState - previousState) * 0.005);
      };
};
```

Reply:
sounds good